File: output/gemma4-e2b/avatar/Java/Python/atcoder_ABC178_A.py
import sys

try:
    # Read the single integer input
    x = int(sys.stdin.read().strip())
except:
    # Handle case where input might be missing or invalid
    sys.exit()

# Java logic translation:
# bool = x == 1
# result = !bool ? 1 : 0
# If x == 1, bool is True, result is 0.
# If x != 1, bool is False, result is 1.

if x == 1:
    result = 0
else:
    result = 1

print(result)

File: output/gemma4-e2b/avatar/Python/Java/atcoder_ABC178_A.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class atcoder_ABC178_A {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        // The Python code reads input inside solve(tc).
        // The loop structure in Python runs only once (tc=1, tcs=1).
        String line = br.readLine();
        
        if (line != null) {
            try {
                // a = int ( stdin.readline ( ).strip ( ) )
                int a = Integer.parseInt(line.trim());
                
                // print ( 1 if a == 0 else 0 )
                System.out.println(a == 0 ? 1 : 0);
            } catch (NumberFormatException e) {
                // Handle potential parsing errors if input is malformed
            }
        }
    }
}